examples/hitl/pick_throw_vr/pick_throw_vr.py

```diff
@@ -105,10 +105,7 @@ def __init__(self, sandbox_service):
 
         self._gui_agent_ctrl.line_renderer = sandbox_service.line_render
 
-        # choose initial value for toggle
-        self._is_remote_active_toggle = (
-            self._sandbox_service.args.remote_gui_mode
-        )
+        self._is_remote_active_toggle = False
```

Comment:
Unrelated to Hydra config migration: one-line fix for how to toggle keyboard/VR control (should always start with local keyboard control).